interview-questions.md

```diff
@@ -9,59 +9,68 @@ Then, research the question to expand on your answer. Even if you feel you have
 
-  Researched answer:
+  Researched answer: The life cycle methods---mounting, updating, unmounting, and error handling----signify the common methods to use during various phases of creating, managing, or removing a React application. The render() method is the only required method in any component class. This method should not modify the component state; it should just display the result of the component it invoked. The render() method should also examine this.state and this.props. Use the other life cycle methods to interact with the browser. In other words, render is for viewing purposes only.
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: JSX is the script style used in React. In other words, the JavaScript language formatted for React. A notable difference between HTML and JSX is that understandings in React when creating classes and divisions (therefore, less keystrokes to make things like a table, paragraphs, etc).
 
-  Researched answer:
+  Researched answer: JavaScript Syntax Extension (JSX) also known as JavaScript XML is some syntactical sugar for React. Developers can place HTML right into JavaScript using JSX. However, one notable difference between JSX and HTML is defining inline styles. Styles within JSX are written as an object with its properties in camelCase and its values in quotations. After that structure is set, you can then pass the object inline. HTML gives more freedom to define as many objects as desired within the opening tag. The value of each object will be in quotations. though this endless structure is available in HTML, it is recommended to keep the list at a minimal to maintain the visibility of the website through search engines. Can't make much of a profit if no one can find your webpage.
 
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: `yarn` is a function within git that allows for certain applications like Jest and React to download dependencies. These dependencies (software/supportive applications) allow Jest and React to pull up projects, open browsers, or perform unit tests.
 
-  Researched answer:
+  Researched answer: Looking for an independent open-source project manager? Then check out yarn. The yarn platform allows you to create a repository that can take on several sub components. It also allows other developers to provide solutions to various problems. Just a giant box of collaboration. Certain yarn commands are very useful with React, such as $ yarn create react-app creates a directory and installs all the necessary files and dependencies. $ yarn start will open a browser for the current project and allow the terminal to be a temporary server. Yarn is certainly a thread to secure an efficient, cohesive working foundation.
```

Comment:
Amazing write up! Please if you don't already have a blog for development you gotta make this post!